python/cudf_polars/tests/test_scan.py

```diff
@@ -13,6 +13,8 @@
     assert_ir_translation_raises,
 )
 
+NO_CHUNK_ENGINE = pl.GPUEngine(raise_on_fail=True, parquet_options={"chunked": False})
+
```

Comment:
*nit*: Should we just test the chunked reader and the non-chunked reader for all parquet tests? I would prefer yes, since the route to applying filters is different (for example).

I.e. parametrize the format with `parquet` and `chunked-parquet` and use that to determined the `"chunked"` parameter.

Reply:
yeah, no problem.

Reply:
Concretely, we'd error if `skip_rows` is passed to `pl.scan_parquet`. But allow `.collect(engine=GPUEngine(parquet_options={'chunked': False, 'skip_rows': 42})`

Reply:
No, `skip_rows` comes from the polars IR, we can't ask the user to pass it in like that.

`skip_rows` is exposed for the chunked reader in pylibcudf and seems to be implemented in the C++. Can you report a bug that it doesn't work?

Reply:
Lawrence put in a fix in https://github.com/rapidsai/cudf/pull/16944/commits/df341ea408c76d29a2456493be9644fbd71abecd.